### PR TITLE
Clarify Pooled Connections vs. Connection Groups. Resolves #956.

### LINKED ISSUE #956: Impl: Pooled Connection
## Pooled Connection {#pooled-connections}

- So implementation talks about pooled connections - I must have blinked and missed that, obviously it is different to a connection group - or is it? If I am asking a silly question then I suspect we need  just a few lines of text to demuddle me.

### PR DESCRIPTION
**Pull request by mwelzl**
Resolves #956.

### CODE REVIEW

**Review by gorryfair — APPROVED** [review]
Seems to answer my issue.

File: draft-ietf-taps-impl.md [hunk 1]
```diff
@@ -738,10 +738,11 @@ errors that can be delivered to the application as Soft Errors. These allow the
...
-For these kinds of protocols, implementations may hide the connection management and only expose a single Connection object and the individual requests/responses as messages.
-These Pooled Connections can use multiple connections or multiple streams of multi-streaming connections between endpoints, as long as all of these satisfy the requirements, and prohibitions specified in the Selection Properties of the Pooled Connection. 
-This enables implementations to realize transparent connection coalescing, connection migration, and to perform per-message endpoint and path selection by choosing among these underlying connections.
+For applications that do not need in-order delivery of Messages, the Transport Services implementation may distribute Messages of a single Connection across several underlying transport connections or multiple streams of multi-streaming connections between endpoints, as long as all of these satisfy the requirements and prohibitions specified via Selection Properties.
```
Comment (philsbln) [inline draft-ietf-taps-impl.md:741]: For me, prohibitions is redundant and just makes reading harder 
```suggestion
For applications that do not need in-order delivery of Messages, the Transport Services implementation may distribute Messages of a single Connection across several underlying transport connections or multiple streams of multi-streaming connections between endpoints, as long as all of these satisfy the requirements specified via Selection Properties.
```

**Review by philsbln — APPROVED** [review]
Works for me and makes the reasoning clearer and less theoretical.
The small change suggestion is optional, thus, improving readability IMHO.

File: draft-ietf-taps-impl.md (hunk 1 above)
Comment (gorryfair) [reply draft-ietf-taps-impl.md:741]: I think you do need to check the prohibitions also?

Comment (abrunstrom) [reply draft-ietf-taps-impl.md:741]: I would think so as well.

**Review by abrunstrom — APPROVED** [review]
LGTM, thanks!

File: draft-ietf-taps-impl.md (hunk 1 above)
Comment (philsbln) [reply draft-ietf-taps-impl.md:741]: Oh … for me "requirements" actually means everything you express by using "require" or "prohibit" … would probably benefit explicitly stating that and making the remainder easier to read.

Comment (gorryfair) [reply draft-ietf-taps-impl.md:741]: I tried a fix in the text - if it doesn't please suggest something else.

Comment (mwelzl) [reply draft-ietf-taps-impl.md:741]: I don't see the fix. But a solution might be: ".... as long as all of these satisfy the specified Selection Properties."

Comment (tfpauly) [reply draft-ietf-taps-impl.md:741]:
```suggestion
For applications that do not need in-order delivery of Messages, the Transport Services implementation may distribute Messages of a single Connection across several underlying transport connections or multiple streams of multi-streaming connections between endpoints, as long as all of these satisfy the Selection Properties.
```

**Review by tfpauly — APPROVED** [review]

**Pull request merged**